user_sync/rules.py

```diff
@@ -44,7 +44,9 @@ def __init__(self, caller_options):
             'remove_user_key_list': None,
             'remove_list_output_path': None,
             'remove_nonexistent_users': False,
-            'default_country_code': None
+            'default_country_code': None,
+            'max_deletions_per_run': None,
```

Comment:
Maybe the default shouldn't be None.  If there is a value always, we don't need None checks.